lib/sendserver.c

```diff
@@ -43,6 +43,72 @@ static int rc_check_reply(AUTH_HDR *, int, char const *, unsigned char const *,
  * @{
  */
 
+/** Encodes CHAP password and stores it in an output buffer 
+ *
+ * @param vp a pointer to a VALUE_PAIR.
+ * @param auth a pointer to AUTH_HDR.
+ * @param id an identifer to use for encryption.
+ * @param output an output buffer to store encrypted chap password string
+ * @return value 0 on success
+ */
+static int rc_chap_encode(VALUE_PAIR * vp, AUTH_HDR * auth, int id, uint8_t * output)
```

Comment:
Given that the `output` is of fixed size I think it is easier to read when the defining output as `char output[MD5_DIGEST_LENGTH]`